app/views/publish/users/show.html.erb

```diff
@@ -10,16 +10,19 @@
           component.row do |row|
             row.key { "First name" }
             row.value(text: @provider_user.first_name, html_attributes: { id: "first_name" })
+            row.action(text: t("change"), href: edit_publish_provider_user_path(provider_code: @provider.provider_code), visually_hidden_text: t("users.first_name").downcase) unless @provider_user.has_sign_in_account?
           end
 
           component.row do |row|
             row.key { "Last name" }
             row.value(text: @provider_user.last_name, html_attributes: { id: "last_name" })
+            row.action(text: t("change"), href: edit_publish_provider_user_path(provider_code: @provider.provider_code), visually_hidden_text: t("users.last_name").downcase) unless @provider_user.has_sign_in_account?
           end
 
           component.row do |row|
             row.key { "Email address" }
             row.value(text: @provider_user.email, html_attributes: { id: "email" })
```

Comment:
What is the use for the id?

Reply:
Removed: https://github.com/DFE-Digital/publish-teacher-training/pull/3185/commits/e0ca3e656408f9c0564e8e98b1329a2f24cd060b